packages/web/src/hooks/useClaimFees.ts

```diff
@@ -38,28 +37,39 @@ export const useClaimFees = (
 ) => {
   const { audiusSdk } = useQueryContext()
   const queryClient = useQueryClient()
+  const { data: currentUser } = useCurrentAccountUser()
 
   return useMutation<ClaimFeesResponse, Error, UseClaimFeesParams>({
     mutationFn: async ({
       tokenMint,
-      ownerWalletAddress,
-      receiverWalletAddress
+      ownerWalletAddress
     }: UseClaimFeesParams): Promise<ClaimFeesResponse> => {
       const sdk = await audiusSdk()
-      await appkitModal.switchNetwork(solana)
       const solanaProvider = appkitModal.getProvider<SolanaProvider>('solana')
       if (!solanaProvider) {
         throw new Error('Missing SolanaProvider')
       }
       if (!ownerWalletAddress) {
         throw new Error('Missing owner wallet address')
       }
+      let splWallet = currentUser?.spl_wallet?.toString()
```

Comment:
sorry, super annoying but i did already add this

https://github.com/AudiusProject/apps/pull/13147

you can opt for your changes and revert mine if you'd like!